### DISCUSSION TITLE
Ventoy scanning files, please wait...

### DISCUSSION THREAD

**Discussion by mvasiliiv** [ventoy/Ventoy#1231] [General]
Good afternoon, it is possible to somehow arrange a process (on the screenshot on the link). On a USB flash drive, many ISO, about 15 and many folders and subfolders, with programs and software.
http://lostpix.com/img/2021-11/12/cp1piygaz3ux9b1k8s598iaba.jpg

**Comment by mvasiliiv** [comment]
I think we need to somehow exclude the Soft folder from scanning and everything else superfluous, so that only folders with iso windows, linux and so on are scanned. Is it possible to implement this somehow?

**Comment by ventoy** [comment]
https://www.ventoy.net/en/doc_search_path.html

**Reply by mvasiliiv** [reply]
Super, everything is set up as it should, now the search is instant, thank you
